languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml

```diff
@@ -887,6 +887,108 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
             </rule>
         </rulegroup>
 
+        <rulegroup id="CONCORDANCIA_DAR_HORAS" name="Concordância de construções como 'deram dez horas' => 'deu dez horas'" default="temp_off">
+            <rule> <!-- deu 10 horas -->
+                <pattern>
+                    <token postag="SENT_START"/> <!-- keep it tightly scoped -->
```

Comment:
In the future, we could add commas and `e` here as well

Reply:
I suppose, but for now I want to keep it extra safe, see how it goes.